src/app/pages/Project.tsx

```diff
@@ -57,7 +104,9 @@ function Docs() {
               <ul key={d.id}>
                 <li>
                   <Link to={`/documents/${d.id}`}>{d.name}</Link>{' '}
-                  <button onClick={() => handleDelete(d.id)}>X</button>
+                  <button onClick={() => handleDelete(d.id, 'single_pack')}>
```

Comment:
I would suggest creating two different function named `handleSinglePackDelete(...)` and `handleMultiPackDelete(...)` instead of passing a flag to one function that do two things.

Reply:
done in the newer commit